api_client/python/timesketch_api_client/client.py

```diff
@@ -414,21 +415,42 @@ def get_aggregator_info(self, name='', as_pandas=False):
 
         return pandas.DataFrame(lines)
 
-    def list_sketches(self):
+    def list_sketches(self, per_page=50, scope='user', include_archived=True):
         """Get a list of all open sketches that the user has access to.
 
-        Returns:
-            List of Sketch objects instances.
+        Args:
+            per_page: Number of items per page when paginating.
```

Comment:
can you add the default values... eg

```
per_page (int): Optional number of items per page when paginating. Defaults to 50.
```

Something like that (and add the type there)

Reply:
done